File: frontEnd/src/types/domain/api/requestModel/MockFieldRuleParams.ts
/**
 * Mock 字段规则参数
 *
 * @deprecated 参数已上移到 MockFieldRule 顶层字段，保留仅用于兼容旧数据。
 */
export class MockFieldRuleParams {
    locale?: string;
    min?: number;
    max?: number;
    scale?: number;
    length?: number;
    format?: string;
    choices?: string;

    constructor() {
        this.locale = 'zh';
        this.min = 0;
        this.max = 100;
        this.scale = 2;
        this.length = 10;
        this.format = 'yyyy-MM-dd HH:mm:ss';
        this.choices = 'A,B,C';
    }
}

File: backEnd/platform-automation/src/main/java/com/mokatest/platform/demos/api/domain/requestModel/MockFieldRuleParams.java
package com.mokatest.platform.demos.api.domain.requestModel;

import lombok.Data;

/**
 * Mock 字段规则参数
 *
 * 已废弃：参数已上移到 {@link MockFieldRule} 顶层字段。保留此类仅用于兼容旧数据反序列化。
 *
 * @author JingLong
 * @since 2026-06-17
 */
@Deprecated
@Data
public class MockFieldRuleParams {

    /**
     * 语言/地区：zh / en
     */
    private String locale;

    /**
     * 最小值（int/float）
     */
    private Integer min;

    /**
     * 最大值（int/float）
     */
    private Integer max;

    /**
     * 小数位数（float）
     */
    private Integer scale;

    /**
     * 长度（text）
     */
    private Integer length;

    /**
     * 日期格式（date）
     */
    private String format;

    /**
     * 选项列表（choice），英文逗号分隔
     */
    private String choices;
}
